eng/config/PublishData.json

```diff
@@ -78,7 +78,6 @@
       "Microsoft.CodeAnalysis.ExternalAccess.OmniSharp": "arcade",
       "Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.CSharp": "arcade",
       "Microsoft.CodeAnalysis.ExternalAccess.AspNetCore": "vs-impl",
-      "Microsoft.CodeAnalysis.ExternalAccess.Razor": "vs-impl",
```

Comment:
we can't remove, needed for servicing branches

Reply:
Thank you